content/docs/learning-guide/tutorials/remote-repositories/create-remote-repository.md

```diff
@@ -0,0 +1,176 @@
+- `--release-archive-path`: the path to where you want to place your .zip archive, we recommend placing it in the Gems folder inside your remote repository folder. 
+- `--download-prefix`: the URI where your gem archive can be download from, You must know where your zip will be avaliable to download before you actually create the release archive zip.
+
+{{< note >}}
+`--release-archive-path` command can only be used after you call the `--add-gem` command.See below example:
+{{< /note >}}
+
+
+```
+o3de.bat edit-repo-properties --repo-path "C:\remote-repo\repo.json" --add-gem "C:\remote-repo\Gems\MyGem" --release-archive-path "C:\remote-repo\Gems" --download-prefix "https://github.com/<YourGitAccount>/<RemoteRepoName>/releases/download/<ReleaseTag>"
+```
+
+1. Here is an example of where you should upload your release to Github (you can find this on your version control URI):
+
+    {{<image-width src="/images/learning-guide/tutorials/remote-repositories/add_release.png" width="500">}}
+
+2. Here is where you would be setting your release tag (make sure this tag is the same as the last parameter you passed for the `--download-prefix` arg):
+
+    {{<image-width src="/images/learning-guide/tutorials/remote-repositories/release_tag.png" width="500">}}
+
+{{< important >}}
+Make sure you know where your release will be available after uploading it to GitHub. Once uploaded, you won't be able to change the `download_source_uri` in your `repo.json` file. If you input the wrong download path, you will need to repeat this step and re-zip the file and upload the release to GitHub again.
+{{< /important >}}
+
+After you upload your Gem to the version control website, other users that have added your remote repository will be able to download it using the Project Manager and `o3de` CLI. Users can navigate to Remote Sources and use your remote project URL to download associated Gems based on the download path you provided in this step.
+
+
+## Testing your remote repo locally
+You can test your remote repo locally before uploading to your remote repository to make sure your remote repo works as expected by using a local file path.
+
+You can test locally be creating another remote repository and Gem using the steps above and add the Gem to your new remote repository.  Next, create a release archive for the new Gem using a local path for the `download prefix`:
+```cmd
+o3de.bat edit-repo-properties --repo-path "C:\local-repo\repo.json" --add-gem "C:\local-repo\Gems\MyLocalGem" --release-archive-path "C:\local-repo\Gems" --download-prefix "C:\local-repo\Gems"
+```
+
+After performing the above steps you should be able to add the new remote repository using the local path `file://c/local-repo` and then be able to download `MyLocalGem` from the `Gems` tab in the Project Manager.
+
+
+#### Use Python to host a `local server` for testing
+
+1. Start a local Python server from the same folder that contains the `repo.json` file.
```

Comment:
Update the wording in this step to describe what the user is actually going to do in the step. If you want to give an overview of what these steps will be doing, then put that in a sentence before the first numbered step. For example, you could have a sentence right before step 1 that says: "Start a local Python server in your local repo folder."
```suggestion
1. Open a command window and change to the local repo folder that contains the `repo.json` file.
```